index.scss

```diff
@@ -0,0 +1,975 @@
+        }
+    }
+}
+
+// <!-- Importent Note End -->
+
+// <!-- Contact Form Start -->
+.contect-section {
+    margin: 100px auto;
+    width: 80%;
+
+    h2 {
+        font-size: $heading-size;
+        line-height: 38px;
+        font-weight: 600;
+        color: $heading-color;
+    }
+
+    p {
+        font-size: $text-size;
+        font-weight: 400;
+        line-height: 26px;
+        color: $primery-color;
+        max-width: 561px;
+    }
+
+    .contect-form-main {
+        gap: 20px;
+        @include flexbox(space-between);
+
+        .cotent-form {
+            button {
+                background-color: $site-color;
+                max-width: 271px;
+                color: #ffff;
+                height: 44px;
+                border-radius: 5px;
+                padding: 10px 25px;
+                font-size: $text-size;
+                font-weight: 600;
+                line-height: 24px;
+                text-align: center;
+                margin-top: 30px;
+                border: none;
+            }
+
+            form {
+                position: relative;
+
+                h6 {
+                    font-size: $text-size;
+                    line-height: 22px;
+                    font-weight: 400;
+                    color: $primery-color;
+                }
+
+                .name-email {
+                    display: flex;
+                    gap: 20px;
+
+                    .user-icon {
+                        position: absolute;
+                        top: 37px;
+                        left: 10px;
+                    }
+
+                    .mail-icon {
+                        position: absolute;
+                        top: 37px;
+                        left: 300px;
+                    }
+
+                    .name-box {
+                        width: 270px;
+                    }
+
+                    .text-field {
+                        width: 270px;
+                        border-radius: 6px;
+                        padding: 8px 12px;
+                        height: 40px;
+                        border: 1px solid #d9d9d9;
+                        color: #7E7287;
+
+                        &::placeholder {
+                            padding-left: 20px;
+                            font-size: $text-size;
+                            line-height: 24px;
+                            font-weight: 400;
+                            color: #7E7287;
+                        }
+
+                        &active-placeholder::placeholder {
+                            color: $site-color;
+                        }
+                    }
+                }
+
+                .massage {
+                    margin-top: 20px;
+
+                    .massage-field {
+                        width: 561px;
+                        height: 140px;
+                        border: 1px solid #d9d9d9;
+                        color: #7E7287;
+                        border-radius: 6px;
+
```

Comment:
width: 561px;
@media screen and max-width(991px) {
  font-size: 400px;
}
@media screen and max-width(575px) {
  font-size: 300px;
}